### ISSUE TITLE
NTR: Bisphosphoglycerate mutase activity low in RBC (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8225]
New term request Bisphosphoglycerate mutase activity low in RBC (elthat_130418182001): Erythrocytes, red blood cells

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000557
- label: Reduced erythrocyte bisphosphoglycerate mutase activity
- parents: [Abnormal erythrocyte enzyme level (HP:0030272)](https://hpo.jax.org/app/browse/term/HP:0030272)
- pmid: PMID:25015942
- def: Activity or concentration of bisphosphoglycerate mutase (BPGM; EC 5.4.2.4;) in red blood cells below the lower limit of normal. BPGM is a multifunctional enzyme specifically found in red blood cells that synthesizes 2,3-diphosphoglycerate through its synthase activity and degrades it through its phosphatase activity.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8225
- synonyms:
